nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -284,6 +284,7 @@ def __init__(self, decoding_cfg, decoder, joint, blank_id: int):
                     decoder_model=decoder,
                     joint_model=joint,
                     blank_index=self.blank_id,
+                    cuda_optimized=self.cfg.get('cuda_optimized', False),
```

Comment:
False as default here, but set to True in dataclass, so the dataclass value will always override this default here when you do change_decoding_strategy. Either set this to True to match dataclass, or update dataclass to match this.